dask/array/tests/test_linalg.py

```diff
@@ -730,9 +759,31 @@ def test_norm_2dim(shape, chunks, axis, norm, keepdims):
 
     # Need one chunk on last dimension for svd.
     if norm == "nuc" or norm == 2 or norm == -2:
-        d = d.rechunk((d.chunks[0], d.shape[1]))
+        rechunkaxis = -1
+        d = d.rechunk({rechunkaxis: -1})
 
     a_r = np.linalg.norm(a, ord=norm, axis=axis, keepdims=keepdims)
     d_r = da.linalg.norm(d, ord=norm, axis=axis, keepdims=keepdims)
 
     assert_eq(a_r, d_r)
+
+
+@pytest.mark.parametrize("shape, chunks, axis", [
+    [(4, 5, 6, 7), (2, 2, 2, 2), (1,2)],
+    [(4, 5, 6, 7), (2, 2, 2, 2), (-1,-2)],
```

Comment:
nit: would be good to have a space after `,`s.

Reply:
I agree. Even somewhat surprised flake8 did not pick this up

Reply:
Yeah, also thought that was weird. @mrocklin, do you have any ideas why flake8 missed this?